lalrpop-util/src/lib.rs

```diff
@@ -232,6 +232,20 @@ pub struct ErrorRecovery<L, T, E> {
     pub dropped_tokens: Vec<(L, T, L)>,
 }
 
+#[cfg(not(target_os = "windows"))]
+#[doc(hidden)]
+#[macro_export]
+macro_rules! path_separator {
+    {} => { "/" }
+}
+
+#[cfg(target_os = "windows")]
```

Comment:
I wonder if we might want to add a windows test run into our testing matrix, just for sanity checking.  As far as I see this is our first `cfg(target_os = "windows")`, although I don't have a particular amount of confidence to say that we don't have some hidden non-portability somewhere, except that I assume someone would have complained if we didn't work on windows.

Reply:
https://github.com/lalrpop/lalrpop/pull/1056